packages/client/hmi-client/src/page/project/components/tera-project-overview.vue

```diff
@@ -61,22 +59,23 @@
 						icon="pi pi-share-alt"
 						class="p-button p-button-secondary quick-link-button"
 					/>
-					<Button
-						label="New workflow"
-						size="large"
-						icon="pi pi-sitemap"
-						class="p-button p-button-secondary quick-link-button"
-					/>
-					<Button
-						label="Compare models"
-						size="large"
-						icon="pi pi-share-alt"
-						class="p-button p-button-secondary quick-link-button"
-					/>
+					<Button size="large" class="p-button p-button-secondary quick-link-button">
+						<vue-feather
+							class="p-button-icon-left"
+							type="git-merge"
+							size="1.25rem"
+							stroke="rgb(16, 24, 40)"
+						/>
+						<span class="p-button-label">New workflow</span>
+					</Button>
+					<Button size="large" class="p-button p-button-secondary quick-link-button">
+						<compare-models-icon fill="rgb(16, 24, 40)" style="scale: 1.25" />
```

Comment:
Because this is an SVG, it might be better to add a class, and style with CSS for both `fill` and `style` attributes.